examples/deno/crate/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
The other examples all have `examples/foo/Cargo.toml`, so could this follow the same pattern?